bigbluebutton-html5/imports/ui/components/external-video-player/service.js

```diff
@@ -17,13 +20,25 @@ const stopWatching = () => {
 };
 
 const sendMessage = (event, data) => {
-  const streamer = getStreamer(Auth.meetingID);
+  const {
+    time,
+    rate,
+    state,
+    timestamp,
+  } = data;
+
+  const meetingId = Auth.meetingID;
+  const userId = Auth.userID;
+
+  const user = Users.findOne({ userId });
+  const suc = user && user.presenter;
```

Comment:
Checking permissions on the client side isn't very secure. The real checks should be on the server-side where it can't be gone around.